service/dashboard/models.py

```diff
@@ -8,6 +8,10 @@
 
 class Dashboard(Model):
 
+    id = CharField(
```

Comment:
As discussed earlier, please move the dashboard and widget models into the api app